doc: add known issue for ISO HCI command. (#14020)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.

> .. rst-class:: v2.6.0

Reviewer: ```suggestion
.. rst-class:: v2-6-0
```

> DRGN-21293: Le Read ISO TX Sync is implemented according to the raised errata ES-23138. 

Reviewer: ```suggestion
DRGN-21293: The LE Read ISO TX Sync command is implemented according to the raised errata ES-23138. 
```

>   In ES-23138, the return parameter TX_Time_Stamp is modified to be the SDU synchronization reference of the SDU previously scheduled for transmission, and not the CIG or BIG reference anchor point.

Reviewer: ```suggestion
  In ES-23138, the return parameter ``TX_Time_Stamp`` is used as the SDU synchronization reference of the SDU previously scheduled for transmission.
  This differs from the Bluetooth Core Specification v5.4, which uses the CIG or BIG reference anchor point as the SDU synchronization reference.
```

>   Note that there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference when the CIG or BIG is configured with an ISO_Interval that equals the SDU_Interval.

Reviewer: ```suggestion
  Note that when the CIG or BIG is configured with an ``ISO_Interval`` that equals the ``SDU_Interval``, there is no difference between the CIG or BIG reference anchor point and the SDU synchronization reference .
```
Alternatively, these could be just ISO interval and SDU interval, and not be parameters at all.

>   If several SDUs are transmitted per ISO_Interval, meaning that the ISO_Interval is larger than the SDU_Interval, our implementation of LE Read ISO TX Sync returns a unique SDU synchronization reference per SDU.

Reviewer: ```suggestion
  If several SDUs are transmitted during each ``ISO_Interval``, meaning that the ``ISO_Interval`` is larger than the ``SDU_Interval``, our implementation of the LE Read ISO TX Sync command returns a unique SDU synchronization reference for each SDU.
```
